satMutMPRA.pm

```diff
@@ -0,0 +1,274 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <https://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl helpdesk at
+<https://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+satMutMPRA - saturation mutagenesis-based massively parallel reporter assays (satMutMPRA) for 21 regulatory elements (11 enhancers, 10 promoters) - Add satMutMPRA data fields to the VEP output
```

Comment:
This is quite long for a name and contains text which would fit better under description.